lib/active_model/serializer/adapter/json.rb

```diff
@@ -1,47 +1,128 @@
 class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
-        extend ActiveSupport::Autoload
```

Comment:
weird that this isn't indented since it is on master https://github.com/rails-api/active_model_serializers/blob/4399c1ad1af2da9dabac327ebf4cb23ff7354e83/lib/active_model/serializer/adapter/json.rb 

need help rebasing?